src/Diagnostics/LogRequestsToDisk.php

```diff
@@ -104,15 +104,12 @@ public function LogPlatformRequests($xml, $url, $headers, $isRequest)
                     $collapsedHeaders[] = "{$key}: {$val}";
                 }
 
-                file_put_contents($filePath,
-                                  ($isRequest?"REQUEST":"RESPONSE")." URI FOR SEQUENCE ID {$sequenceNumber}\n==================================\n{$url}\n\n",
-                                  FILE_APPEND);
-                file_put_contents($filePath,
-                                  ($isRequest?"REQUEST":"RESPONSE")." HEADERS\n================\n".implode("\n", $collapsedHeaders)."\n\n",
-                                  FILE_APPEND);
-                file_put_contents($filePath,
-                                  ($isRequest?"REQUEST":"RESPONSE")." BODY\n=============\n".$xml."\n\n",
-                                  FILE_APPEND);
+                $what = $isRequest?"REQUEST":"RESPONSE";
+                $content = "$what URI FOR SEQUENCE ID $sequenceNumber\n==================================\n$url\n\n
+                            $what HEADERS\n================\n".implode("\n", $collapsedHeaders)."\n\n".
+                           "$what BODY\n=============\n".$xml."\n\n";
+                file_put_contents($filePath, $content);
```

Comment:
Thanks for simplifying! You missed the last argument `FILE_APPEND` for the method `file_put_contents()`.

Reply:
That's was my intention. We don't need it since we always create a new file here